earthpy/tests/test_clip.py

```diff
@@ -1,96 +1,46 @@
 import shapely
 import geopandas as gpd
 import earthpy.clip as cl
 
 
-""" Setup dummy data for tests """
-
-pts = np.array([[2, 2],
-                [3, 4],
-                [9, 8],
-                [-12, -15]])
-
-locs = [Point(xy) for xy in pts]
-locs_gdf = gpd.GeoDataFrame(locs,
-                            columns=['geometry'],
-                            crs={'init': 'epsg:4326'})
-
-# TODO? Create a create gdf from list function since we do it like 3
-# times in this test file already
-
-
-""" Create Line Objects For Testing """
-
-linea = LineString([(1, 1), (2, 2), (3, 2), (5, 3)])
-lineb = LineString([(3, 4), (5, 7), (12, 2), (10, 5), (9, 7.5)])
-
-linez_gdf = gpd.GeoDataFrame([1, 2],
-                             geometry=[linea, lineb],
-                             crs={'init': 'epsg:4326'})
-
-""" Create Polygon Objects For Testing """
-
-# Create crop box
-poly_inters = Polygon([(0, 0), (0, 10), (10, 10), (10, 0), (0, 0)])
-poly_in_gdf = gpd.GeoDataFrame([1],
-                               geometry=[poly_inters],
-                               crs={'init': 'epsg:4326'})
-
-# Shift by 20
-poly_out_gdf = poly_in_gdf.copy()
-poly_out_gdf = poly_in_gdf.geometry.apply(lambda x: shapely.affinity.
-                                          translate(x, xoff=20, yoff=20))
-# Buffer points to create multi poly
-locs_buff = locs_gdf.copy()
-locs_buff["geometry"] = locs_gdf.buffer(4)
-
-# Create geom w "donut hole"
-donut_geom = gpd.overlay(locs_buff, poly_in_gdf,
-                         how='symmetric_difference')
-
-multi_poly = donut_geom.unary_union
-multi_gdf = gpd.GeoDataFrame(gpd.GeoSeries(multi_poly),
-                             crs={'init': 'epsg:4326'})
-multi_gdf = multi_gdf.rename(columns={0: 'geometry'}).\
-    set_geometry('geometry')
-
-
-
-""" Run clip shape tests """
-
-
-def test_returns_gdf():
+def test_returns_gdf(locs_gdf, poly_in_gdf):
     """Test that function returns a GeoDataFrame (or GDF-like) object."""
     out = cl.clip_shp(locs_gdf, poly_in_gdf)
     assert hasattr(out, 'geometry')
 
 
 def test_non_overlapping_geoms():
     """Test that a bounding box returns error if the extents don't overlap"""
+    unit_box = Polygon([(0, 0), (0, 1), (1, 1), (1, 0), (0, 0)])
+    unit_gdf = gpd.GeoDataFrame([1],
+                                geometry=[unit_box],
+                                crs={'init': 'epsg:4326'})
+    non_overlapping_gdf = unit_gdf.copy()
+    non_overlapping_gdf = unit_gdf.geometry.apply(lambda x: shapely.affinity.
+                                                  translate(x, xoff=20))
     with pytest.raises(ValueError):
-        cl.clip_shp(locs_gdf, poly_out_gdf)
+        cl.clip_shp(unit_gdf, non_overlapping_gdf)
 
 
-def check_input_gdfs():
+def check_input_gdfs(poly_in_gdf):
     """Test that function fails if not provided with 2 GDFs."""
     with pytest.raises(AssertionError):
-        cl.clip_shp(locs, poly_in_gdf)
+        cl.clip_shp([], poly_in_gdf)
```

Comment:
just curious. i know this is passing an empty list. is this good form over calling `list()` ? it seems like a mistake when i look at it but i know what you are doing. would `list()` be more explicit?

Reply:
Changed to `list()`